components/script/script_runtime.rs

```diff
@@ -248,6 +250,35 @@ unsafe extern "C" fn empty(extra: *const c_void) -> bool {
     result
 }
 
+#[allow(unsafe_code)]
+unsafe extern "C" fn push_new_interrupt_queue(interrupt_queues: *mut c_void) -> *mut c_void {
+    let mut result = std::ptr::null_mut();
+    wrap_panic(&mut || {
+        let interrupt_queues = &mut *(interrupt_queues as *mut Vec<Rc<MicrotaskQueue>>);
+        let mut new_queue = Rc::new(MicrotaskQueue::default());
+        // TODO: SAFETY: is this ok? It subverts the Rc::get_mut() check
```

Comment:
With my suggestions from https://github.com/servo/mozjs/pull/595#pullrequestreview-3048546030 I don't believe get_mut is necessary here.